### ISSUE TITLE
Ability to add a exclude_paths to .readable.json

### ISSUE THREAD

**Issue by SemProvoost** [officient/readable#9]
'exclude_paths' should overrule 'paths'
e.g.
```
{
  "baseline": ".readable-baseline.json",
  "paths": [
    "server/"
  ],
  "exclude_paths": [
    "server/dir_to_be_excluded/"
  ]
}
```

**Comment by denys-potapov** [comment]
Ok, just a note on syntax. Probably we can use like gitignore config?

```
{
  "baseline": ".readable-baseline.json",
  "paths": [
    "server/",
    "!server/dir_to_be_excluded/"
  ]
}
```

**Comment by denys-potapov** [comment]
Fixed in 31e26d97320a6fa828f0a481c7bc8800dcc1e076
